backend/backend/properties.py

```diff
@@ -254,3 +269,34 @@ def _is_in_monopoly(self):
         if result[0] == 1:
             is_monopoly = True
         return is_monopoly
+
+    def rails_owned(self):
+        """
+        Returns:
+            int: how many railroads are owned by the owner of self.
+        """
+        with self._conn.cursor() as cursor:
+            cursor.execute('SELECT COUNT(*) FROM `properties` ',
+                           'WHERE `game_id` = %s ',
+                           'AND `property_type` = `railroad` ',
+                           'AND `player_id` = %s',
+                           (self._gid, self._owner))
+            result = cursor.fetchone()
+            return result[0] - 1
+
+    def utils_owned(self):
+        """
+        Returns:
+            int: how many utilities are owned by the owner of self.
+        """
+        multiplier = 1
+        with self._conn.cursor() as cursor:
+            cursor.execute('SELECT COUNT(*) FROM `properties` ',
+                           'WHERE `game_id` = %s ',
+                           'AND `property_type` = `utility` ',
+                           'AND `player_id` = %s',
+                           (self._gid, self._owner))
+            result = cursor.fetchone()
+            if result[0] == 2:
```

Comment:
The pymysql cursor we're using is DictCursor, so `result` needs to be indexed by **key** rather than **int**. In this case you would do something like:
`if result["count(*)"] == 2:`

Reply:
I thought if there was only one column that would work. Looked it up there and changing it to result.keys()[0] instead of just 0.